src/SFA.DAS.EAS.Employer_Account.Database/SFA.DAS.EmployerApprenticeshipsService.Database.refactorlog

```diff
@@ -1,107 +1,121 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Has anything genuinely changed in this file? If not probably best to reset to previous commit, so it drops out of diff